ParisBNF/abb/BNFabb88.xml

```diff
@@ -36,62 +36,66 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <locus from="5r" to="178r"/>
                             <title ref="LIT7229DossierWP"/>
                             <msItem xml:id="ms_i1.1">
```

Comment:
as I had written in the previous PR you need a generic `title `for 1.1 ("Life and Miracles of WP")